plugins/hadoop-authorization-sentry-default/hadoop-authorization-sentry/src/main/java/com/thinkbiganalytics/datalake/authorization/config/SentryConfiguration.java

```diff
@@ -44,37 +46,65 @@
+                                                              , @Value("${beeline.userName}") String userName
+                                                              , @Value("${beeline.password}") String password
+                                                              , @Value("${hdfs.hadoop.configuration}") String hadoopConfiguration
+                                                              , @Value("${authorization.sentry.type}") String authorizationGroupType
+                                                              , @Value("${authorization.sentry.unix.group.filePath}") String linuxGroupFilePath
+                                                              , @Value("${authorization.sentry.ldap.url}") String ldapURL
+                                                              , @Value("${authorization.sentry.ldap.authDn}") String ldapAdmin
+                                                              , @Value("${authorization.sentry.ldap.password}") String ldapPassword
+                                                              , @Value("${authorization.sentry.ldap.authenticator.groupDnPatterns}") String groupDnPattern
+                                                              , @Value("${authorization.sentry.groups}") String sentryGroups
+                                                              , @Value("${sentry.kerberos.principal}") String kerberosPrincipal
+                                                              , @Value("${sentry.kerberos.KeytabLocation}") String kerberosKeytabLocation
+                                                              , @Value("${sentry.IsKerberosEnabled}") String kerberosEnabled) {    
         SentryConnection sentryConnection = new SentryConnection();
         sentryConnection.setDriverName(driverURL);
         sentryConnection.setSentryGroups(sentryGroups);
         sentryConnection.setHadoopConfiguration(hadoopConfiguration);
         sentryConnection.setDataSource(dataSource(connectionURL, driverURL, userName, password));
         sentryConnection.setKerberosTicketConfiguration(createKerberosTicketConfiguration(kerberosEnabled, hadoopConfiguration, kerberosPrincipal, kerberosKeytabLocation));
+        sentryConnection.setAuthorizationGroupType(authorizationGroupType); 
+        sentryConnection.setLinuxGroupFilePath(linuxGroupFilePath);
+        sentryConnection.setLdapContextSource(getLdapContextSource(ldapURL,ldapAdmin,ldapPassword));
+        sentryConnection.setLdapGroupDnPattern(groupDnPattern);
 
         SentryAuthorizationService hadoopAuthorizationService = new SentryAuthorizationService();
         hadoopAuthorizationService.initialize(sentryConnection);
         return hadoopAuthorizationService;
     }
 
     public DataSource dataSource(String connectionURL
-        , String driverURL
-        , String userName
-        , String password) {
+                                 , String driverURL
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "driverURL". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)